docker-compose.dev.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+
+services:
+  postgres:
+    image: postgres
+    restart: always
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: strapi
+      POSTGRES_PASSWORD: strapi
+      POSTGRES_DB: strapi
+    ports:
+      - '5432:5432'
+
+  mongo:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
```

Comment:
Btw, just asking, but is there a reason why the mongo default username is different than the other databases?

Reply:
I think there is one but I don't remember it right now 🤔 I will verify

Reply:
Ok just tested and it breaks the connection 🤷‍♂ I won't spend too much time on the topic ;)

Reply:
👍 🤷‍♂